Homepage

Rizwan-learn opened the pull request:
From The previous Pull request, https://github.com/Rizwan-learn/Hello_Microverse/pull/1, I have made amends. 

- Linters has been updated from .github/workflows/.github/workflows/linters.yml to .github/workflows/linters.yml
- Node_modules has been added to .gitignore 
- ReadMe File has been updated to make it look more professional and easier to read.

Rizwan-learn committed:
Added a .gitignore for the Homepage branch 

.gitignore has been added to the homepage

Rizwan-learn committed:
Created Linters 

HTML and CSS Linters has been added to Homepage Branch

Rizwan-learn committed:
Updated ReadMe

ReadMe has been updated to better explain what the project

Rizwan-learn committed:
Index.html

Basic html create and added a H1

Rizwan-learn committed:
CSS has been added

The h1 colour was changed to red

Rizwan-learn committed:
Updated Gitignore

test.md has been added to the git ignore

Rizwan-learn committed:
New Linters

Previous linter had failed, new linters have been added.

Rizwan-learn committed:
update Linters

Linters has been updated as it was having an error.

Rizwan-learn committed:
updated Gitignore

Node_modules has been added to Git ignore

Rizwan-learn committed:
Linters Updates

This has been added to fix linters errors for WebHint

Rizwan-learn committed:
Linters Stylellint update

This has been added to resolve Stylehint errors when requesting a pull request.

Rizwan-learn committed:
File created

File create to move new linters location

Rizwan-learn committed:
Updated ReadMe file

ReadMe file has been updated and made easier to read

FrederickMih commented on .github/workflows/.github/workflows/linters.yml:
```diff
@@ -0,0 +1,48 @@
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@6.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
- [ ] Hi @Rizwan-learn,
It seems you haven't yet resolved the linters issues raised by the previous code reviewer. There's still the `.github/workflows` directory inside the main `.github/workflows` directory. You may consider removing it.

- [ ] Once more, let's double-check this [Link](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up linters correctly

Rizwan-learn replied:
Hi @FrederickMih , I have removed the linters within the workflow. however, I'm not sure if there are any errors Can you let me know if there is anything wrong with my linters?

FrederickMih commented on README.md:
```diff
@@ -1 +1,60 @@
+_______________________________________________________________________________
+
+
+Additional description about the project and its features.
+
+Built With
+Major languages
+Frameworks
+Technologies used
+
+_______________________________________________________________________________
+Live Demo
+Live Demo Link
+_______________________________________________________________________________
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+
+Setup
+
+Install
+
+Usage
+
+Deployment
```
- [ ] As pointed out by the previous code reviewer, you need to personalize these details in the template not by just listing them. For example
**Setup**: Clearly explain how another developer can set up this project in his/her local machine.
**Built With**: Explain the major languages you have used to build this project.
**etc.**

FrederickMih reviewed (changes requested):
Hi @Rizwan-learn , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Rizwan-learn committed:
Deleting Hello_Microverse / .github / workflows / .github / workflows /

Deleting Hello_Microverse/.github/workflows/.github/workflows/ as it was created in the wrong location, new Linters has been added.
